galaxy_ng/tests/integration/api/test_artifact_download.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import subprocess
+import tempfile
+from unittest.mock import patch
+
+import pytest
+from orionutils.generator import build_collection, randstr
+
+from ..constants import USERNAME_PUBLISHER
+from ..utils import (
+    CapturingGalaxyError,
+    CollectionInspector,
+    get_client,
+    set_certification,
+    wait_for_task
+)
+
+logger = logging.getLogger(__name__)
+
+
+def test_download_artifact(ansible_config, upload_artifact):
+    config = ansible_config("partner_engineer")
```

Comment:
recommend testing this with the basic user, since you don't need any permissions to download

Reply:
good catch!